src/sign-in-wpf/MsalExample.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>WinExe</OutputType>
+		<TargetFramework>net6.0-windows</TargetFramework>
+		<Nullable>enable</Nullable>
+		<UseWPF>true</UseWPF>
+		<ImplicitUsings>disable</ImplicitUsings>
+	</PropertyGroup>
+
+	<ItemGroup>
+	  <None Remove="AzureAD.png" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Identity.Client" Version="4.*" />
+	</ItemGroup>
+
+	<ItemGroup>
+	  <Resource Include="AzureAD.png" />

```

Comment:
It's a nice touch and I hate to even request it, but we should avoid including any asset not directly related to the teaching task. Logos are especially problematic because they change, then they're out of date, then... You get the idea. 🙁

Reply:
Totally understand.  Will remove.

Reply:
Removed in https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/26/commits/2356d2a8a387d27cf1d848f2cea872a67e327ae9